lib/ProductOpener/Users.pm

```diff
@@ -210,7 +210,7 @@ sub display_user_form($$$) {
 	. textfield(-name=>'email', -value=>$user_ref->{email}, -size=>80, -autocomplete=>'email', -type=>'email', -override=>1) . "</td></tr>"
 	. "\n<tr><td>$Lang{username}{$lang}<br/><span class=\"info\">" . (($type eq 'edit') ? '': $Lang{username_info}{$lang}) . "</span></td><td>"
 	. (($type eq 'edit') ? $user_ref->{userid} :
-		( textfield(-id=>'userid', -name=>'userid', -value=>$user_ref->{userid}, -size=>40, -onkeyup=>"normalize_string_value(this)", -autocomplete=>'username'))) . "</td></tr>"
+		( textfield(-id=>'userid', -name=>'userid', -value=>$user_ref->{userid}, -size=>40, -onkeyup=>"normalize_string_value(this)", -autocomplete=>'username' -oninput='this.value = this.value.replace(/[^a-zA-Z0-9 -]/, '')'))) . "</td></tr>"
```

Comment:
The issue is that Chrome fills this field with an email address. (see https://github.com/openfoodfacts/openfoodfacts-server/issues/4968 )
so we get values like someusername@gmail.com for instance. We want to reject those values entirely, here we would just remove the @ character.
Maybe a solution is to check if the value contains @, and if it does, we completely delete the value.